ui/components/app/nfts-tab/nfts-tab.js

```diff
@@ -65,6 +67,23 @@ export default function NftsTab() {
               <NFTsDetectionNoticeNFTsTab />
             </Box>
           ) : null}
+          {(process.env.MULTICHAIN && hasAnyNfts === true) ||
+          process.env.MULTICHAIN === false ? null : (
+            <Box
+              paddingInlineStart={4}
+              paddingInlineEnd={4}
+              display={Display.Flex}
+            >
+              <AssetListConversionButton
+                variant="nft"
+                onClick={() =>
+                  global.platform.openTab({
+                    url: 'https://support.metamask.io/hc/en-us/articles/360058238591-NFT-tokens-in-your-MetaMask-wallet',
```

Comment:
Nit: Can we store this url in a constant?

Reply:
Yep!